gpflow/optimizers/mcmc.py

```diff
@@ -41,41 +41,34 @@ class SamplingHelper:
 
         @tf.function
         def run_chain_fn():
-            return mcmc.sample_chain(num_samples, num_burnin_steps, current_state, kernel=adaptive_hmc)
+            return mcmc.sample_chain(
```

Comment:
was this formatting done with black and line-length 80?

Reply:
I don't think so. `black` doesn't touch docstring (as far as I know)